pom.xml

```diff
@@ -1518,36 +1525,6 @@
         <artifactId>maven-enforcer-plugin</artifactId>
       </plugin>
       <plugin>
-        <groupId>org.codehaus.mojo</groupId>
```

Comment:
This used to have a good purpose. Earlier versions of scala-maven-plugin were not adding the sources. However it was added a year ago or so, definitely in the version spark is using.